src/components/userAccount/userAccount.js

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable react/destructuring-assignment */
```

Comment:
- [ ] As it is now, Webpack won't compile unless `userAccount.css` is present.  I had to add the file in order to run the project.  Kindly correct this issue.

![image](https://user-images.githubusercontent.com/57421823/193006656-3daa05fd-da52-491c-b32f-8f5e64ffc828.png)

Reply:
- I just needed to add this script again, because at this point the project doesn't have it. Please fix this bug. 🧇